bin/check-vault-status.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+#
+# check-vault-status
+#
+# DESCRIPTION:
+#   This plugins checks if vault is up & reachable. It then checks
+#   if the vault is unsealed
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: vault
+#
+require 'sensu-plugin/check/cli'
+require 'vault'
+
+class VaultStatus < Sensu::Plugin::Check::CLI
```

Comment:
Hmm I am wondering if this should be called `VaultSealed` (and rename the file to reflect) as its not the overall health/status we are checking only that vault is sealed or not.